check-vulnerabilities/action.yml

```
@@ -377,7 +386,13 @@ runs:
 
         # Run security tools
         safety check -o bare --save-json info_safety.json --continue-on-error $ignored_vulnerabilities
-        bandit -r "${SOURCE_DIRECTORY}" -o info_bandit.json -f json --exit-zero
+
+        if [[ "$BANDIT_CONFIGFILE" == "" ]]; then
+          CONFIGFILE=""
+        else
+          CONFIGFILE="-c $BANDIT_CONFIGFILE"
+        fi
+        bandit $CONFIGFILE -r "${SOURCE_DIRECTORY}" -o info_bandit.json -f json --exit-zero

```

Just to make sure variables expand properly:

```suggestion
        if [[ "${BANDIT_CONFIGFILE}" == "" ]]; then
          CONFIGFILE=""
        else
          CONFIGFILE="-c ${BANDIT_CONFIGFILE}"
        fi
        bandit ${CONFIGFILE} -r "${SOURCE_DIRECTORY}" -o info_bandit.json -f json --exit-zero

```

---

Thanks for the review and for that suggested change!